<launch>
 <param name="root_frame" value="world" />
 <param name="child_frame" value="odom" />

    <group>
        <node pkg="first_project" name="gps_to_odom" type="gps_to_odom"/>

        <node pkg="first_project" name="odom_to_tf" type="odom_to_tf">
            <param name="child_frame" value="wheel_odom" />
            <remap from="input_odom" to="gps_odom"/>
            <remap from="input_odom" to="odom"/>

        </node>
    </group>
    
</launch>